### DISCUSSION TITLE
Make bookmark for Users page

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#8760] [Feature Requests]
It will be convinient to use bookmarks for special filtering user's collections i.e. for moderators.

### Directus version 
9.0.0.rc-96

**Comment by thefairman** [comment]
has this feature not been added yet?

**Reply by rijkvanzanten** [reply]
We prioritize new features and improvements based on community interest and feedback, and as you can tell, this one hasn't really taken off in the last year.. Feel free to open a PR though if you'd like to see this!

**Reply by rijkvanzanten** [reply]
Scratch that, looks like @j0hnfl0w already did just now 🙂  https://github.com/directus/directus/pull/15187

**Reply by IlyaPetrovM** [reply]
Awesome!

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
